services/brig/schema/src/V18.hs

```diff
@@ -12,7 +12,7 @@ migration = Migration 18 "Add prekeys" $ do
     void $ schema' [r|
         create columnfamily if not exists clients
             ( user   uuid
-            , client ascii
+            , client text
```

Comment:
should we put these changes in a separate PR maybe?

Reply:
On thinking about it again, that's probably best, yes.